Dockerfile

```diff
@@ -12,31 +12,35 @@ RUN apt-get update \
     && docker-php-ext-install -j$(nproc) mysqli snmp \
     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
     && docker-php-ext-install -j$(nproc) gd \
+    # Development extensions
+    && pecl install xdebug-2.5.5 \
```

Comment:
Is the reference to a particular xdebug version intentional? 
It does prevent problems when there is an incompatible update, but could cause the same if any of the other unversioned references receives an incompatible update

Reply:
As far as I know, this is the standard way of installing xDebug. In pecl (https://pecl.php.net/package/xdebug) there is no "latest" for the binaries.
Furthermore, with this, we can run controlled upgrades and upgrades need a PR as well.

Reply:
According to https://xdebug.org/docs/install there is `pecl install xdebug`.
We could either leave it at the most current version (and then fix issues with a PR when or if they ever appear) or fix all of the packages and then take care about updates manually. As this requires a certain discipline and regularity, I would go for number one, given that those packages should be fairly stable and to save maintenance effort